sdk/python/setup.py

```diff
@@ -137,7 +106,6 @@ def find_version(*file_path_parts):
     entry_points={
         'console_scripts': [
             'dsl-compile = kfp.deprecated.compiler.main:main',
-            'dsl-compile-v2 = kfp.compiler.main:main',
-            'kfp=kfp.__main__:main'
+            'dsl-compile-v2 = kfp.compiler.main:main', 'kfp=kfp.__main__:main'
```

Comment:
tip: adding a comma after the last item of a list would give uniform formatting.